src/cpu.c

```diff
@@ -128,6 +139,28 @@ int clockCpu(struct CPU *cpu, struct BUS *bus) {
       case STY:
         writeBus(address, cpu->index_y, bus);
         break;
+      case NOP:
+        printf("let's slide!");
+        break;
+      case TXA: 
+        cpu->index_x = cpu->accumulator;
```

Comment:
Not sure whether the accumulator needs to be zero'ed after the operation or not...

Reply:
No it remains as it is